external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -2418,6 +2430,19 @@ impl<'a> ParsingSymbolicator<'a> {
         }
     }
 
+    fn attr_symbols(&mut self, sp!(_, attr): P::Attribute) {
+        use P::Attribute_ as A;
+        match attr {
+            A::Name(_) => (),
+            A::Assigned(_, v) => {
+                update_cursor!(self.cursor, *v, Attribute);
+            }
+            A::Parameterized(_, sp!(_, attributes)) => {
+                attributes.iter().for_each(|a| self.attr_symbols(a.clone()))
+            }
+        }
```

Comment:
I'm a little confused about this. In general, I would think we would call `update_cursor` at the _top_ of this function.